integration-cli/docker_cli_daemon_test.go

```diff
@@ -780,6 +780,18 @@ func (s *DockerDaemonSuite) TestDaemonDefaultGatewayIPv4Explicit(c *check.C) {
 	deleteInterface(c, defaultNetworkBridge)
 }
 
+func (s *DockerDaemonSuite) TestDaemonDefaultGatewayIPv4ExplicitOutsideContainerSubnet(c *check.C) {
```

Comment:
Issue #14859, besides the original one, tracks a [second issue](https://github.com/docker/docker/issues/14859#issuecomment-125124552).

Fix for the former issue is already present in Docker master, and this PR adds a test to cover that.
Fix for the second issue is part of this vendoring. Test code for it is in libnetwork.

So it is expected this integ test already passes on master.